client/src/components/RoundAddDialog.vue

```diff
@@ -88,6 +89,10 @@ export default {
       type: String,
       required: true,
     },
+    gameTeam: {
+      type: Object,
+      required: true
+    }
```

Comment:
Could you remove those changes about coop, as they related to the different PR

Reply:
yes sorry it was by mistake. I need to find how to remove it now. I am really noob at PR i just started learning.

Reply:
you can try to revert this commit 1fbf4f26f3e99deaa8f19ae11f08127f362f3b46
or remove those changes manually